packages/terra-form-checkbox/tests/jest/CheckboxField.test.jsx

```diff
@@ -8,12 +8,30 @@ import Checkbox from '../../src/Checkbox';
 
 window.matchMedia = () => ({ matches: true });
 
+let userAgentGetter;
+beforeEach(() => {
+  userAgentGetter = jest.spyOn(window.navigator, 'userAgent', 'get');
+});
+
 it('should render a default checkbox field', () => {
   const checkBox = (<CheckboxField legend="Default CheckboxField" />);
   const wrapper = shallowWithIntl(checkBox);
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render checkbox field with div element for safari browser or Edg browser', () => {
+  userAgentGetter.mockReturnValue('safari Edg');
+  const wrapper = shallowWithIntl(<checkboxField legend="Coustom Message CheckboxField" />);
```

Comment:
To get the inner contents of the component:
```suggestion
  const wrapper = shallowWithIntl(<checkboxField legend="Coustom Message CheckboxField" />).dive();
```